graphql-api/fragments/DocumentDetails.graphql

```diff
@@ -0,0 +1,41 @@
+fragment DocumentDetails on Document {
```

Comment:
Please put these files right next to the vue files that use them, and maybe rename them, so that its relatively easy to jump to the correct graphql file.

Reply:
I'm not sure about this, but wouldn't it conflict with the react-router if we put graphql files in pages?